### PR TITLE
Kosovo

### PR DESCRIPTION
**Pull request by Vesihiisi**
Mapping: https://www.wikidata.org/wiki/Wikidata:WikiProject_WLM/Mapping_tables/xk_(sq)
Preview: https://www.wikidata.org/wiki/Wikidata:WikiProject_WLM/Mapping_tables/xk_(sq)/preview

Task: https://phabricator.wikimedia.org/T176111

### CODE REVIEW

File: importer/XkSq.py [hunk 1]
```diff
@@ -1,62 +1,81 @@
...
+
+    def update_labels(self):
+        albanian = utils.remove_markup(self.name)
+        self.add_label("sq", albanian)
+
+    def update_descriptions(self):
+        english = "cultural heritage monument of Kosovo"
+        self.add_description("en", english)
+
+    def exists_with_monument_article(self, language):
```
Comment (lokal-profil) [inline importer/XkSq.py]: think this can be skipped as it's standard field and mapping language

Comment (Vesihiisi) [reply importer/XkSq.py]: :heavy_check_mark:

**Comment by Vesihiisi** [comment]
Added a basic mapping for category :arrow_right: P31.

**Comment by Vesihiisi** [comment]
And yes the branch is called Albania for stupid reasons :/

**Comment by Vesihiisi** [comment]
Preview up to here.

File: importer/XkSq.py [hunk 2]
```diff
@@ -1,62 +1,96 @@
...
+        municip_matches = utils.get_matching_items_from_dict(
+            municip_raw, municp_dic)
+        if len(municip_matches) == 1:
+            self.add_statement("located_adm", municip_matches[0])
+        else:
+            self.add_to_report(
+                "municipality", self.municipality, "located_adm")
 
-        The values are never wikilinked.
-        E panjohur = unknown
```
Comment (lokal-profil) [inline importer/XkSq.py]: This line would still be nice to have (since I encountered it in the previews)

Comment (lokal-profil) [reply importer/XkSq.py]: Just spotted this in the online mapping. Maybe set it to "-" there to make it visible. Then no need to keep it here.

**Comment by Vesihiisi** [comment]
Corrected municipality lookup after [two items were redirected since the initial mapping](https://www.wikidata.org/w/index.php?title=Wikidata%3AWikiProject_WLM%2FMapping_tables%2Fxk_%28sq%29%2Fmunicipalities&type=revision&diff=587875719&oldid=435234314).

**Comment by lokal-profil** [comment]
@Vesihiisi Artanë as well

**Pull request merged**